package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "pretest": "webdriver-manager update --gecko false",
     "test": "protractor test/protractor.conf.js",
-    "prettify": "prettier --write **/*.js **/*.md package.json sampleApp/**/* --print-width 120"
```

Comment:
Why removing it from markdown files? Just curious.

Reply:
I verified now and did not see a good reason for that. If you want to go back to previous mode.

Reply:
I think that would be nice.

Reply:
Done.